meetup/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from pizzaplace.models import PizzaPlace
+from django.core.validators import RegexValidator
+from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from django.core.exceptions import ValidationError
+
+
+def validate_urlname(link):
+  validator = RegexValidator(
+    regex='meetup\.com\/\w+(-\w+)*\/$',
+    message="Does not conform to Meetup Url",
+    code='invalid_url')
+  return validator(link)
+
+def validate_meetup_exists(link):
+  looker = MeetupApiLookupAgent(link)
+  is_real = looker.is_real_meetup()
+
+  if not is_real:
```

Comment:
I think it's OK to inline the "is real meetup" check instead of storing the result of `looker.is_real_meetup()`in the `is_real` variable. I see why you did it, though: giving `looker.is_real_meetup()` a name makes this read more naturally. `if not looker.is_real_meetup()` sounds a little weird. I wonder if `looker` is the thing that's weirdly named here, though. What if this was something like `if not MeetupAPI.meetup_exists(link)`? The intention there is pretty clear.